apps/admin-gui/src/app/vos/components/application-form-list/application-form-list.component.html

```diff
@@ -96,11 +96,12 @@
                   value="{{'VO_DETAIL.SETTINGS.APPLICATION_FORM.CLEAR_SELECTION' | translate}}" />
               </form>
             </div>
-            <div
-              *ngIf="applicationFormItem.type ==='HEADING' ||
-                      applicationFormItem.type === 'HTML_COMMENT'">
+            <div *ngIf="applicationFormItem.type ==='HEADING'">
               <span [innerHTML]="getLocalizedLabel(applicationFormItem)"></span>
             </div>
+            <div *ngIf="applicationFormItem.type === 'HTML_COMMENT'">
+              <span [innerHTML]="getLocalizedLabel(applicationFormItem) | sanitizeHtml"></span>
```

Comment:
From what I understand, `HTML_COMMENT` is any html the user inputs, is it safe to call `bypassSecurityTrustHtml` here?

Reply:
Yes, I already discussed it with the Product Owner and the main problem is a few steps before displaying it in GUI when we allow saving potentially unsafe HTML to the database. We will figure it out in the next Sprint(s) and we should solve it on the front end and also on the back end. But the PO said that we can display it at this moment when we already have the HTML in the database.

Reply:
Ok, seems good then